Creating calculator components

tomasesquivelgc opened the pull request:
## In this branch:

 - Calculator component is created.
 - Button child component is created on the same file as Calculator.
 - CSS is added to resemble the template

tomasesquivelgc committed:
boiler plate erase and calculator create

tomasesquivelgc committed:
Calculator css add

tomasesquivelgc committed:
calculator buttons css add

tomasesquivelgc committed:
linters fix

tomasesquivelgc committed:
minor change to update github.com

Shisui6 commented on src/components/calculator.js:
```diff
@@ -0,0 +1,53 @@
+function Calculator() {
+  return (
+    <div className="calculator">
+      <div className="calculator-display">0</div>
```
- Kindly follow the design guidelines for the display by giving it a background color of `#858694`
- [Optional] It would be nice if you centered your calculator so it doesn't stick to the edge

Shisui6 reviewed (changes requested):
Hi @tomasesquivelgc, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- Calculator component created ✅


## Required Changes ♻️

- Please remove the img at `public/logo192.png` as it is a boilerplate file
- Please remove the img at `public/logo512.png` as it is a boilerplate file

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

- For future commits please always remember to start your commits with a capital letter and a verb as this is considered best practise for making commits

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tomasesquivelgc committed:
Change style

Background color of calculator change to match template.
calculator div center
public/logo images delete

binhussen reviewed (approved):
Hi @tomasesquivelgc , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
<img src="https://user-images.githubusercontent.com/65852621/133696182-6c5b784b-0083-427a-b24d-3ec870915f79.jpg"/>
## Highlights
- Well Structured Code✔️
- Fixed Request Changes ✔️
-  Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

binhussen reviewed (approved):
Hi @tomasesquivelgc , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
<img src="https://user-images.githubusercontent.com/65852621/133696182-6c5b784b-0083-427a-b24d-3ec870915f79.jpg"/>
## Highlights
- Well Structured Code✔️
- Fixed Request Changes ✔️
-  Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
